examples/glasbey-colormap.py

```diff
@@ -0,0 +1,81 @@
+"""
+Glasbey colormap
+================
+
+A long-requested feature [1]_ for napari was to display labels/segmentation
+layers using a well-known discrete colormap such as Glasbey [2]_.
+
+In this example, we demonstrate displaying segmentations using custom colormaps
+with the help of the glasbey Python library [3]_. We display a segmentation
+using the napari built-in labels colormap, the original Glasbey colormap, a
+more modern version produced by limiting the lightness and chroma and
+optimizing for colorblind-safety, and finally with the matplotlib tab10
+colormap.
+
+.. [1] https://github.com/napari/napari/issues/454
+.. [2] Colour displays for categorical images. Chris Glasbey, Gerie van der
+       Heijden, Vivian F. K. Toh, and Alision Gray. (2007)
+       DOI:10.1002/col.20327
+.. [3] https://github.com/lmcinnes/glasbey
+
+.. tags:: layers, visualization-basic
+"""
+import glasbey
+from skimage import data
+from skimage.filters import threshold_otsu
+from skimage.measure import label
+from skimage.morphology import closing, remove_small_objects, square
```

Comment:
```suggestion
from skimage.morphology import closing, remove_small_objects, footprint_rectangle
```
`square` triggers a deprecation warning